src/crush/crush.h

```diff
@@ -415,6 +418,12 @@ struct crush_map {
          */
 	__u8 chooseleaf_stable;
 
+	/*! Sets total descents for MSR rules */
+	__u8 msr_descents;
+
+	/*! Sets local collision retries for MSR rules */
+	__u8 msr_collision_tries;
```

Comment:
Is `__u8` large enough?

Reply:
No, that's a mistake.  Good catch, I appreciate it.